src/@next/Tooltip/Tooltip.stories.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
```

Comment:
you have to update the docs with the sample code on how to use the tooltip

Reply:
done